tests/rptest/services/redpanda.py

```diff
@@ -254,13 +254,16 @@ class SISettings:
     The defaults are for use with the default minio docker container, these
     settings are altered in RedpandaTest if running on AWS.
     """
+    GLOBAL_DEDICATED_NODES_KEY = "dedicated_nodes"
+    GLOBAL_CRED_SOURCE_KEY = "cred_source"
     GLOBAL_S3_ACCESS_KEY = "s3_access_key"
     GLOBAL_S3_SECRET_KEY = "s3_secret_key"
     GLOBAL_S3_REGION_KEY = "s3_region"
 
     def __init__(self,
                  *,
                  log_segment_size: int = 16 * 1000000,
+                 cloud_storage_credentials_source: str = 'config_file',
```

Comment:
i'm assuming the [documented](https://docs.redpanda.com/docs/platform/reference/cluster-properties/#cloud-storage) default value of `config_file` is safe to preserve existing default behavior of the ducktape tests

Reply:
from conversation with @abhijat earlier today, i'm going to remove this line and also line 283 so that the default behavior will work

Reply:
this comment is obsolete